packages/react-components/src/components/Select/Select.tsx

```diff
@@ -86,13 +90,32 @@ function ChevronUp() {
   );
 }
 
+/* Icon displayed when input is in invalid state */
```

Comment:
Keep this here for now and we can replace it with a common file in a new PR when more of these outstanding ones are merged and it's clear where it's needed.